.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst, README.rst
```

Comment:
@sils1297 it won't IIRC. There's no `/` in the path to the README. The better option would be to use `**.rst`